app/overrides/decorate_admin_configurations_index.rb

```diff
@@ -1,4 +1,6 @@
-Deface::Override.new(virtual_path: 'spree/admin/shared/sub_menu/_configuration',
-                     name: 'add_stores_to_admin_menu',
-                     insert_bottom: '[data-hook="admin_configurations_sidebar_menu"]',
-                     text: '<%= configurations_sidebar_menu_item Spree.t(:stores_admin), spree.admin_stores_url %>')
+Deface::Override.new(
```

Comment:
`app/views/spree/admin/shared/sub_menu/_configuration.html.erb` already contains the "stores" menu. Delete this file